python/mxnet/gluon/block.py

```diff
@@ -445,6 +445,14 @@ class HybridBlock(Block):
     the end-to-end usage.
     """
     def __init__(self, prefix=None, params=None):
+        # check if any parameter is row_sparse
+        if isinstance(params, ParameterDict):
+            for param in params.values():
+                stype = param._stype
+                if stype != 'default':
+                    raise ValueError("Cannot create a HybridBlock with Parameter '%s' " \
+                                     "because its storage type is %s. Please consider " \
+                                     "using a SparseBlock instead."%(param.name, stype))
```

Comment:
PR for sparse block will be created separately after this one is merged.

Reply:
"please consider using" -> "please use"